src/ik_evolution_1.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2016-2017, Philipp Sebastian Ruppel
```

Comment:
Where did this file come from? Do we know this person?

Reply:
Made so we can call `makeEvolution2Solver` elsewhere. Philipp is the original author of the codebase